src/stores/proposals.ts

```diff
@@ -19,7 +19,8 @@ const PROPOSALS_LIMIT = 20;
 export const useProposalsStore = defineStore('proposals', {
   state: () => ({
     proposals: {} as Partial<Record<string, SpaceRecord>>,
-    summaryProposals: {} as Partial<Record<string, SpaceRecord>>
+    summaryProposals: {} as Partial<Record<string, SpaceRecord>>,
+    executionTemp: null as any
```

Comment:
We should define type if we can also I'm not sure we need a new state variable we could just leverage the draft system to create a new draft when someone confirm a transfer from Treasury page and redirect the user to that draft.